src/components/FetchTrial.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+class ListProducers extends Component{
+    constructor(){
+        super();
+        this.state = {
+            producers: []
```

Comment:
why do you have an internal state in this component when you have a redux store?